air-quality-backend/system_tests/api_suite/measurements_api_seeded_data_tests.py

```diff
@@ -141,7 +190,7 @@
 location_names_test_city_3 = "Test City 3"
 
 # Test Setup
-load_dotenv(".env-qa")
+load_dotenv()
```

Comment:
FYI we can probably just move this to the top of the test files and call it once rather than include in every test. Happy if you'd rather leave for now though

Reply:
Changes made as suggested